aem/aem_console_setup.md

```diff
@@ -1,36 +1,76 @@
 
 
-### Add the `I/O Events` API to your Adobe developer console project
-
-
-For your AEM to be authorized to emit events against Adobe I/O, you need to add the `I/O Events` API to your project in the Adobe developer console. If you are new to this, please refer to the [step-by-step instructions for creating an empty project in Adobe Developer Console](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/projects-empty.md) )
+### Add the `I/O Events` and `I/O Management API` API to your Adobe developer console workspace
```

Comment:
Why both? 
I/O Management API -> For registering the provider using v2 APIs
I/O Events API -> For emitting events
Is the above understanding correct?

Reply:
yes we need both